introduction-editor.html

```diff
@@ -95,15 +97,15 @@ <h3 id="gabc">Creating a gabc score file</h3>
 <p>Copy the above into a new file and save it as <code>test.tex</code> in the same directory as <code>example.gabc</code></p>
 
 <h3 id="compile">Compiling the document</h3>
-<p>The goal now is to compile this file using your TeX editor to produce a PDF output.  Select <code>LuaLaTeX+se</code> from the drop down list in the upper left, or automate this selection by placing the following line at the begining of the file:</p> 
+<p>The goal now is to compile this file using your <span class="tex">T<span class="epsilon">e</span>X</span> editor to produce a PDF output.  Select <code>LuaLaTeX+se</code> from the drop down list in the upper left, or automate this selection by placing the following line at the begining of the file:</p> 
 
 <div class="commandline">
-<code>% !TEX TS-program = LuaLaTeX+se</code>
+<code>% !TEX program = LuaLaTeX+se</code>
 </div>
 
 <p>Then click on the "Typeset" button.</p>
 
-<p>If you correctly followed the instructions of this page, a PDF file will show up in the right part of your screen!  You're now ready to start playing with your document file by using <a href="/gregoriotex/details.html">the various GregorioTeX commands available to tweak the appearence of the score.</a></p>
+<p>If you correctly followed the instructions of this page, a PDF file will show up in the right part of your screen!  You're now ready to start playing with your document file by using <a href="gregoriotex/details.html">the various Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> commands available to tweak the appearence of the score.</a></p>
```

Comment:
To me, the sounds like the PDF will appear in the correct part of the screen.  I think perhaps this should say `on the right size of your screen` rather than `in the right part of your screen`.